src/common/modules/navigations/gnb/modules/gnb-menu/modules/dashboard-recent-favorite/modules/GNBDashboardFavorite.vue

```diff
@@ -177,14 +196,24 @@ export default {
                 state.loading = false;
             }
         };
+
+        const isOverflown = () => {
+            if (state.listContainerRef && state.favoriteRef) {
+                return (state.favoriteRef?.clientHeight < state.listContainerRef?.scrollHeight);
+            } return false;
+        };
+
         /* Init */
         (async () => {
             state.loading = true;
             await Promise.allSettled([
                 getFavoriteOrderList(),
                 store.dispatch('favorite/load', FAVORITE_TYPE.DASHBOARD),
+                store.dispatch('dashboard/loadDomainDashboard'),
+                store.dispatch('dashboard/loadProjectDashboard'),
             ]);
             state.loading = false;
+            nextTick(() => emit('update:is-overflown', isOverflown()));
```

Comment:
nitpick: Please use async/await rather than promise callback.

```suggestion
            await nextTick()
            emit('update:is-overflown', isOverflown());
```

Reply:
issue: When the favorite list are updated, overflow state will be change. 
I cannot find the code for this situation. 🥲

Reply:
I will add it after connecting to api. Thank you for the check!